browser_history/browsers.py

```diff
@@ -91,6 +120,68 @@ def _deeper(array, folder, bookmarks_list):
                     )
         return bookmarks_list
 
+    def cookies_parser(self, cookie):
+
+        """Returns cookies of a single input for Chrome based browsers
+        The returned datetimes are timezone-aware with the local timezone set
+        by default
+
+        :param cookie : dictionary containing cookie details
+        :type cookie : dict
+        :return a tuple of cookie information
+        :rtype: list(tuple(:py:class: str, str, `datetime.datetime`, str, str, str))
+        """
+        platform = utils.get_platform()
+        if EXC is not None:
+            utils.logger.error(EXC)
```

Comment:
Instead of just printing the error, add a line saying which packages have to be installed.

Reply:
Also, exiting here is bad. Raise an exception instead.
Exiting on error should be done in the CLI.

Reply:
Okay sure

Reply:
What's the split for?
Also, we should mention the packages names there (from `cookie-requirements.txt`) like so: 
```... module not found. The following packages are required for retrieving cookies on Chromium-based browsers: "keyring", "pbkdf2", "pyaes".``` (this would change according to the platform)

Reply:
split was to get the package name, 

> we should mention the packages names there (from `cookie-requirements.txt`) like so:
> `... module not found. The following packages are required for retrieving cookies on Chromium-based browsers: "keyring", "pbkdf2", "pyaes".` (this would change according to the platform)

I'll do this,sounds like a better option